contrib/src/main/java/com/datatorrent/contrib/hive/AbstractFSRollingOutputOperator.java

```diff
@@ -183,12 +184,7 @@ protected void rotateCall(String lastFile)
     try {
       this.rotate(lastFile);
     }
-    catch (IOException ex) {
-      logger.debug(ex.getMessage());
-      DTThrowable.rethrow(ex);
-    }
-    catch (ExecutionException ex) {
-      logger.debug(ex.getMessage());
+    catch (IOException | ExecutionException ex) {
```

Comment:
The HiveMockTest also has a issue. the  fsRolling.setRotationWindows should be called to set the window. 
And the HiveMockTest is using the embed Hive, you in fact can test even if Hive not installed.